analyses/cnv-comparison/util/cnv-comparison-functions.R

```diff
@@ -0,0 +1,200 @@
+# This script defines filtering and plotting functions to be sourced in 
+# `01-cnv-comparison-plotting.R`
+#
+# Chante Bethell for CCDL 2019
+# 
+# Usage:
+# This script is intended to be run via the command line from the top directory 
+# of the repository as follows:
+#
+# Rscript analyses/cnv-comparison/util/cnv-comparison-functions.R
+
+read_in_cnv <- function(input_directory, file_path){
+  # Given the file path of the CNV data, read in the data.
+  #
+  # Arg:
+  #   input_directory: file path of input directory
+  #   file_path: file path of input data
+  #
+  # Return:
+  #   dataframe: the data frame containing the input data
+
+  # Read in cnv data
+  dataframe <-
+    read.table(gzfile(file.path(input_directory, file_path)), header = TRUE)
+
+  # Rename the columns
+  dataframe <-
```

Comment:
I now realize that my comment above this line was misleading as this was meant to rearrange the columns, rather than simply rename them. That being said, the function `colnames` won't be feasible here. I am thinking of using `dplyr::select` as the replacement here unless either of you may have a better solution.

Reply:
Yeah, do `dplyr::select/rename`